src/React.Web/AspNetCache.cs

```diff
@@ -28,10 +28,9 @@ public class AspNetCache : ICache
 		/// <summary>
 		/// Initializes a new instance of the <see cref="AspNetCache"/> class.
 		/// </summary>
-		/// <param name="context">The HTTP context</param>
-		public AspNetCache(HttpContextBase context)
+		public AspNetCache()
 		{
-			_cache = context.Cache;
+			_cache = HttpRuntime.Cache;
```

Comment:
I'd prefer if this was injected via the dependency injection container, to more easily facilitate unit testing. Could you register this  `AssemblyRegistration.cs`? Something like:
```
container.Register<Cache>((c, o) => HttpRuntime.Cache);
```

Reply:
I'd prefer if this was injected via the dependency injection container, to more easily facilitate unit testing. Could you register this  `AssemblyRegistration.cs`? Something like:
```
container.Register<Cache>((c, o) => HttpRuntime.Cache);
```

Then inject it into the constructor:
```
public AspNetCache(Cache cache)
```